### PR TITLE
Various jobs start with more job-related programs preloaded into their PDAs.

### PR DESCRIPTION
**Pull request by Vincent983**
## About The Pull Request

Engineers now start with the canary and atmosscan program, atmos techs start with the SM monitor program, and Robotisists start with the silliconnect program.

## Why It's Good For The Game
it's more convinient than downloadin em roundstart.

## Changelog
:cl:
qol: Various jobs have gotten additional programs uploaded at roundstart, engineers now have the canary and atmospheric scanner programs, atmospheric technicians now have the supermatter monitor, and Robotisists/RDs now have the silliconnect program.

/:cl:

### CODE REVIEW

**Review by JohnFulpWillard — CHANGES_REQUESTED** [review]
Atmos techs should have supermatter monitor

Command intentionally does not have card monitor because they each have ID management consoles in their offices.

**Comment by Vincent983** [comment]
oh, alright, by the way can I add atmozphere to the engineer PDA as well?

**Comment by lessthnthree** [comment]
can HoP please get Plexagon Access Management as well?

**Comment by the-orange-cow** [comment]
It might be prudent to specify which jobs get which apps in your change log.

**Comment by Jackal-boop** [comment]
I think you can mark this as QOL

**Comment by Vincent983** [comment]
fair

**Comment by carlarctg** [comment]
Give the bounty program to assistants - it was originally meant to but the author didn't know how that worked, and it will let assistants assist.

**Comment by Vincent983** [comment]
@carlarctg which bounty program? I'm lookin at the code and they appear to have the bounty board program preloaded, unless is it broken?

**Comment by carlarctg** [comment]
> @carlarctg which bounty program? I'm lookin at the code and they appear to have the bounty board program preloaded, unless is it broken?

I suppose it 'is' broken, then, or probably more accurately needs to be accessed and 'networked' first for bounties to show up, rather than it being a passive 'New bounty!' thing like it should be. Shame, and out-of-scope for this PR. Ignore me unless you want to put extra effort into this.

File: code/modules/modular_computers/computers/item/disks/role_disks.dm [hunk 1]
```diff
@@ -121,13 +121,16 @@
...
+		/datum/computer_file/program/atmosscan,
+		/datum/computer_file/program/alarm_monitor,
 	)
 
 /obj/item/computer_disk/atmos
 	name = "atmospheric technician data disk"
 	desc = "Removable disk used to download atmos-related tablet apps."
 	icon_state = "datadisk6"
 	starting_programs = list(
+		/datum/computer_file/program/supermatter_monitor,
```
Comment (JohnFulpWillard) [inline code/modules/modular_computers/computers/item/disks/role_disks.dm:133]: This is now just the same as the engineering data disk. Remove one and keep the other

**Comment by JohnFulpWillard** [comment]
> oh, alright, by the way can I add atmozphere to the engineer PDA as well?

yea sure idk its just an analyzer

**Comment by dieamond13** [comment]
I think scientists should also get the science research hub too

**Comment by Vincent983** [comment]
god dammit

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm [hunk 2]
```diff
@@ -95,6 +95,7 @@
 		/datum/computer_file/program/status,
 		/datum/computer_file/program/science,
 		/datum/computer_file/program/robocontrol,
+		/datum/computer_file/program/borg_monitor,
 		/datum/computer_file/program/budgetorders,
```
Comment (san7890) [inline code/modules/modular_computers/computers/item/role_tablet_presets.dm:99]: alphabetize this list

File: code/modules/modular_computers/computers/item/disks/role_disks.dm [hunk 3]
```diff
@@ -116,18 +116,12 @@
...
 		/datum/computer_file/program/supermatter_monitor,
-	)
-
-/obj/item/computer_disk/atmos
-	name = "atmospheric technician data disk"
-	desc = "Removable disk used to download atmos-related tablet apps."
-	icon_state = "datadisk6"
-	starting_programs = list(
 		/datum/computer_file/program/atmosscan,
 		/datum/computer_file/program/alarm_monitor,
```
Comment (san7890) [inline code/modules/modular_computers/computers/item/disks/role_disks.dm:125]: alphabetize

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm [hunk 4]
```diff
@@ -156,13 +157,16 @@
 	greyscale_colors = "#D99A2E#69DBF3#E3DF3D"
 	starting_programs = list(
 		/datum/computer_file/program/supermatter_monitor,
+		/datum/computer_file/program/atmosscan,
```
Comment (san7890) [inline code/modules/modular_computers/computers/item/role_tablet_presets.dm:160]: alphabetize

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm [hunk 5]
```diff
@@ -156,13 +157,16 @@
...
+		/datum/computer_file/program/alarm_monitor,
 	)
 
 /obj/item/modular_computer/pda/atmos
 	name = "atmospherics PDA"
 	greyscale_config = /datum/greyscale_config/tablet/stripe_thick
 	greyscale_colors = "#EEDC43#00E5DA#727272"
 	starting_programs = list(
+		/datum/computer_file/program/supermatter_monitor,
 		/datum/computer_file/program/atmosscan,
```
Comment (san7890) [inline code/modules/modular_computers/computers/item/role_tablet_presets.dm:170]: alphabetize

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm [hunk 6]
```diff
@@ -176,6 +180,7 @@
 	greyscale_config = /datum/greyscale_config/tablet/stripe_thick
 	greyscale_colors = "#FAFAFA#000099#B347BC"
 	starting_programs = list(
+		/datum/computer_file/program/science,
 		/datum/computer_file/program/atmosscan,
```
Comment (san7890) [inline code/modules/modular_computers/computers/item/role_tablet_presets.dm:184]: alphabetize

File: tools/UpdatePaths/Scripts/77740_delete_atmos_disk.txt [hunk 7]
```diff
@@ -0,0 +1,3 @@
+/obj/item/computer_disk/atmos : @DELETE
```
Comment (san7890) [inline tools/UpdatePaths/Scripts/77740_delete_atmos_disk.txt:1]: this helps thanks

**Review by san7890 — CHANGES_REQUESTED** [review]
you can get an extension for vscode that helps you alphabetize with a simple key-command, no reason to do it manually

**Comment by nukularpower** [comment]
nt frontiers for scientists please

**Comment by Vincent983** [comment]
there's a disk in the toxins lab that has NT frontiers on it, just slot it into your PDA

**Review by Ghommie — APPROVED** [review]

**Pull request merged**